sriov/tests/SR_IOV_BondVF_DPDK/test_SR_IOV_BondVF_DPDK.py

```diff
@@ -0,0 +1,155 @@
+import pytest
+from time import sleep
+from sriov.common.utils import (
+    create_vfs,
+    set_vf_mac,
+    execute_and_assert,
+    bind_driver,
+    prepare_ping_test,
+    get_pci_address,
+    start_tmux,
+    wait_tmux_testpmd_ready,
+    stop_tmux,
+    stop_testpmd_in_tmux,
+)
+
+
+class Bond:
```

Comment:
It this the only test case related to bond interfaces? If not, or we don't know yet, I would move class definition into common code.